SignalAnalysis/localization/strings.hi.resx

```diff
@@ -0,0 +1,512 @@
+  <data name="strGrpCulture" xml:space="preserve">
+    <value>यूआई और डेटा प्रारूप</value>
+  </data>
+  <data name="strLblData" xml:space="preserve">
+    <value>डेटा-फ़ाइल पथ</value>
+  </data>
+  <data name="strLblDataFormat" xml:space="preserve">
+    <value>संख्यात्मक डेटा-स्वरूपण स्ट्रिंग</value>
+  </data>
+  <data name="strLblEnd" xml:space="preserve">
+    <value>ऐरे इंडेक्स एंड</value>
+  </data>
+  <data name="strLblSeries" xml:space="preserve">
+    <value>श्रृंखला का चयन करें</value>
+  </data>
+  <data name="strLblStart" xml:space="preserve">
+    <value>सरणी अनुक्रमणिका प्रारंभ</value>
+  </data>
+  <data name="strLblWindow" xml:space="preserve">
+    <value>खिड़की</value>
+  </data>
+  <data name="strMsgBoxErrorFFT" xml:space="preserve">
+    <value>FFT की गणना करते समय अनपेक्षित त्रुटि।
+{0}</value>
+  </data>
+  <data name="strMsgBoxErrorFFTTitle" xml:space="preserve">
+    <value>एफएफटी त्रुटि</value>
+  </data>
+  <data name="strMsgBoxErrorOpenData" xml:space="preserve">
+    <value>डेटा फ़ाइल खोलते समय एक अनपेक्षित त्रुटि हुई।
+कृपया बाद में पुन: प्रयास करें या सॉफ़्टवेयर इंजीनियर से संपर्क करें।
+{0}</value>
+  </data>
+  <data name="strMsgBoxErrorOpenDataTitle" xml:space="preserve">
+    <value>डेटा खोलने में त्रुटि</value>
+  </data>
+  <data name="strMsgBoxErrorSaveData" xml:space="preserve">
+    <value>डेटा फ़ाइल सहेजते समय एक अनपेक्षित त्रुटि हुई.
+Please try again later or contact the software engineer.
+{0}</value>
+  </data>
+  <data name="strMsgBoxErrorSaveDataTitle" xml:space="preserve">
+    <value>डेटा सहेजने में त्रुटि</value>
+  </data>
+  <data name="strMsgBoxExit" xml:space="preserve">
+    <value>क्या आप वाकई ऐप्लिकेशन से बाहर निकलना चाहते हैं?</value>
+  </data>
+  <data name="strMsgBoxExitTitle" xml:space="preserve">
+    <value>बाहर निकलना?</value>
+  </data>
+  <data name="strMsgBoxInitArray" xml:space="preserve">
+    <value>'InitializeDataArrays' में अनपेक्षित त्रुटि।
+{0}</value>
+  </data>
+  <data name="strMsgBoxInitArrayTitle" xml:space="preserve">
+    <value>गलती</value>
+  </data>
+  <data name="strMsgBoxNoData" xml:space="preserve">
+    <value>सहेजे जाने के लिए कोई डेटा उपलब्ध नहीं है।</value>
+  </data>
+  <data name="strMsgBoxNoDataTitle" xml:space="preserve">
+    <value>कोई डेटा नहीं</value>
+  </data>
+  <data name="strMsgBoxOKSaveData" xml:space="preserve">
+    <value>डेटा को डिस्क में सफलतापूर्वक सहेज लिया गया है।</value>
+  </data>
+  <data name="strMsgBoxOKSaveDataTitle" xml:space="preserve">
+    <value>डेटा सहेजा गया</value>
+  </data>
+  <data name="strMsgBoxTaskCancel" xml:space="preserve">
+हॉसडॉर्फ-बेसिकोविच फ्रैक्टल आयाम की गणना रोक दी गई है।</value>
+  </data>
+  <data name="strMsgBoxTaskCancelTitle" xml:space="preserve">
+    <value>विराम</value>
+  </data>
+  <data name="strOpenDlgFilter" xml:space="preserve">
+    <value>ErgoLux files (*.elux)|*.elux|SignalAnalysis files (*.sig)|*.sig|Text files (*.txt)|*.txt|Binary files (*.bin)|*.bin|All files (*.*)|*.*</value>
```

Comment:
Is this OK in Hindi?

Reply:
Those are file types and extensions. In my opinion It's better that they remain in English.

Reply:
All right.